app/models/ticket.rb

```diff
@@ -1,26 +1,28 @@
 class Ticket < ApplicationRecord
   belongs_to :ticket_desk
   belongs_to :cinema_hall
-  belongs_to :client
+  belongs_to :movie
+  
   validate :id, :if => :exist?
-  validates :client_id, numericality: { only_integer: true }
   validates :ticket_desk_id, numericality: { only_integer: true }
   validates :cinema_hall_id, numericality: { only_integer: true }
+  validates :movie_id, numericality: { only_integer: true }
   validates :paid, presence: true, inclusion: { in: [true, false] }
 
-
-  def exist?
-      Ticket.exists?(id: self.ticket_id)
-  end
-
-  def ticket_id
-    self.id
-  end
-
-  def ticket_paid
-    self.paid
-  end
-
-
   
-end
+    def exist?
+        Ticket.exists?(id: self.ticket_id)
+    end
+  
+    def ticket_id
+      self.id
+    end
+  
+    def ticket_paid
+      self.paid
+    end
```

Comment:
`ticket_` here sounds redundat. Also Ruby prefers question marks for such methods, like:

```ruby
def paid?
  paid
end
```